doc/man3/EVP_PKEY_CTX_ctrl.pod

```diff
@@ -284,6 +293,10 @@ generation to I<pubexp>. Currently it should be an odd integer. The
 I<pubexp> pointer is used internally by this function so it should not be
 modified or freed after the call. If not specified 65537 is used.
 
+EVP_PKEY_CTX_set1_rsa_keygen_pubexp() does the same as
+EVP_PKEY_CTX_set_rsa_keygen_pubexp() except that it internally copies the
+I<pubexp>.
```

Comment:
I think it would be better to describe the preferred function `EVP_PKEY_CTX_set1_rsa_keygen_pubexp`, and then describe how the deprecated function `EVP_PKEY_CTX_set_rsa_keygen_pubexp` differs from it.